internal/hcs/process.go

```diff
@@ -359,46 +373,59 @@ func (process *Process) Stdio() (stdin io.Writer, stdout, stderr io.Reader) {
 
 // CloseStdin closes the write side of the stdin pipe so that the process is
 // notified on the read side that there is no more data in stdin.
-func (process *Process) CloseStdin(ctx context.Context) error {
+func (process *Process) CloseStdin(ctx context.Context) (err error) {
+	operation := "hcs::Process::CloseStdin"
+	ctx, span := trace.StartSpan(ctx, operation) //nolint:ineffassign,staticcheck
```

Comment:
Do we still want to make new spans for these?

Reply:
I added them for consistency with gcs and co, which all have spans on their close; but likely no.
But we can address removing them wholesale in the tracing PR
or just remove them here...